metricbeat/docs/autodiscover-kubernetes-config.asciidoc

```diff
@@ -34,6 +34,9 @@ Pods share an identical host. If only the `{data.host}` variable is interpolated
 then one config will be generated per host. The configs will be identical.
 After they are de-duplicated, only one will be used.
 
+In order to target one specific exposed port `{data.host}:{data.port.web}` can be used
+in template config, where `web` is the name of the exposed container port.
+
```

Comment:
Can we add some test for this? It could be done in a follow up if you prefer, I couldn't find tests for `data.port` so we may need to add more things.